cmake/FindOpenVX.cmake

```diff
@@ -25,6 +25,21 @@ endif()
 
 if(OPENVX_INCLUDE_DIR AND OPENVX_LIBRARIES)
   set(HAVE_OPENVX TRUE)
+
+  set(_fname "${CMAKE_CURRENT_BINARY_DIR}/test_ovx_refname.cpp")
+  file(WRITE "${_fname}" "#include <VX/vx.h>\nint main() { return VX_REFERENCE_COUNT == VX_REFERENCE_TYPE ? VX_REFERENCE_NAME : 0; }\n")
```

Comment:
I believe it is better to create this file [here](https://github.com/opencv/opencv/tree/master/cmake/checks) and use it.

You may refer to [this implementation](https://github.com/opencv/opencv/blob/3.2.0/cmake/OpenCVDetectOpenCL.cmake#L20-L29)